server/src/main/java/org/spine3/server/Entity.java

```diff
@@ -50,6 +59,8 @@
 
     private int version;
 
+    private final M defaultState;
```

Comment:
Isn't this state a part of the class, rather than each instance? 

It's immutable and the same for all the instances of the class. The method `getDefaultState()` can and probably should read this state once it's available, but why have it in every instance?